lib/sqlalchemy/ext/mutable.py

```diff
@@ -752,6 +754,12 @@ def _listen_for_type(mapper: Mapper[_T], class_: type) -> None:
 
 _setup_composite_listener()
 
+class _NoArg(Enum):
+    NO_ARG = 0
+
+
+_NO_ARG = _NoArg.NO_ARG
```

Comment:
We already have this somewhere else, not sure if we should reuse it. 

thoughs @zzzeek ?

Reply:
Yeah that’s where I stole it from 🫣, I wasn’t sure if I should use something kinda buried in the `sql` package in `ext.mutable`.

Reply:
Hmm, this also doesn’t let me use `Literal[_NO_ARG]` as a type hint…

Reply:
>  I wasn’t sure if I should use something kinda buried in the sql package

Probably better to create a new one.

Sooner or later we should do a cleanup to optimize a bit the import time, since with all the inline typing it's probably ballooned quite a bit

Reply:
Ahh: https://github.com/python/typing/issues/689#issuecomment-561425237

Reply:
working with python typing is a recipe for frustration sadly, especially after working with typescript

Reply:
I finally used the one from `sql.base` because it’s used elsewhere, too.